README.md

```diff
@@ -132,42 +132,43 @@ Example: Train.jsonl
 
 ## Supported Models
 
-Current supported and tested models are `Granite v3`, `Llama3` (8B configuration has been tested) and `GPTBigCode`.
-
 - For each tuning technique, we run testing on a single large model of each architecture type and claim support for the smaller models. For example, with QLoRA technique, we tested on granite-34b GPTBigCode and claim support for granite-20b-multilingual.
 
 - LoRA Layers supported : All the linear layers of a model + output `lm_head` layer. Users can specify layers as a list or use `all-linear` as a shortcut. Layers are specific to a model architecture and can be specified as noted [here](https://github.com/foundation-model-stack/fms-hf-tuning?tab=readme-ov-file#lora-tuning-example)
 
 - Legend:
 
-✅ Ready and available 
+  ✅ Ready and available 
 
-✔️ Ready and available - compatible architecture (*see first bullet point above)
+  ✔️ Ready and available - compatible architecture (*see first bullet point above)
 
-🚫 Not supported
+  🚫 Not supported
 
-?  may be supported, but not tested
+  ? May be supported, but not tested
 
 Model Name & Size  | Model Architecture | Full Finetuning | Low Rank Adaptation (i.e. LoRA) | qLoRA(quantized LoRA) | 
 -------------------- | ---------------- | --------------- | ------------------------------- | --------------------- |
-Granite 3B           | Granite v3       | ✅*             | ✅*                              | ✅*                   | 
-Granite 8B           | Granite v3       | ✅*             | ✅*                              | ✔️                     | 
-[Granite 3B](https://huggingface.co/ibm-granite/granite-3b-code-base) | LlamawithCausalLM          | ✅ | ✔️  | ✔️ | 
-[Granite 8B](https://huggingface.co/ibm-granite/granite-8b-code-base) | LlamawithCausalLM          | ✅ | ✅ | ✅ |
-Granite 13B                                                           | GPTBigCodeForCausalLM      | ✅ | ✅ | ✔️  | 
-[Granite 20B](Granite-20b-multilingual-base)                          | GPTBigCodeForCausalLM      | ✅ | ✔️  | ✔️  | 
-[Granite 34B](https://huggingface.co/ibm-granite/granite-34b-code-instruct/tree/main) | GPTBigCodeForCausalLM | 🚫 | ✅ | ✅ | 
-Llama3.1-8B                         | LLaMA 3.1          | ✅ - supported from platform up to 8k context length - same architecture as llama3-8b  | ✔️ | ✔️ |  
+Granite 3.0 3B       | GraniteForCausalLM     | ✅* | ✅* | ✅* |
+Granite 3.0 8B       | GraniteForCausalLM     | ✅* | ✅* | ✔️  |
```

Comment:
Since we are not mentioning the power models then the correct family of granite3 models are:
> IBM's Granite 3.0 family includes:
> - General Purpose/Language: Granite 3.0 8B Instruct, Granite 3.0 2B Instruct, Granite 3.0 8B Base, Granite 3.0 2B Base
> - Guardrails & Safety: Granite Guardian 3.0 8B, Granite Guardian 3.0 2B
> - Mixture-of-Experts: Granite 3.0 3B-A800M Instruct, Granite 3.0 1B-A400M Instruct, Granite 3.0 3B-A800M Base, Granite 3.0 1B-A400M Base

So to match this list 
Granite 3.0 2B       | GraniteForCausalLM     | ✔️* | ✔️* | ✔️* |
Granite 3.0 8B       | GraniteForCausalLM     | ✅* | ✅* | ✔️  |


unless we are mentioning the power models in which:
Granite PowerLM 3B       | GraniteForCausalLM     | ✅* | ✅* | ✅* |
Granite 3.0 8B       | GraniteForCausalLM     | ✅* | ✅* | ✔️  |